src/sync.cpp

```diff
@@ -4,13 +4,12 @@
 
 #include <sync.h>
 
+#include <set>
```

Comment:
Why?

Reply:
Amazingly enough the #include tree in sync.cpp did not pull in std::set already.